docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -341,183 +341,34 @@ Now that you have set up your robot and are able to control your motor, you can
-
-#### Add the `MoistureSensor` Remote
-
-Now, go back to your robot's page on [the Viam app](https://app.viam.com) and navigate to the **Config** tab, then to the **Remotes** subtab.
-
-Add your sensor server as a [remote part](/manage/parts-and-remotes/) called `my-sensor-server`:
-
-{{< tabs >}}
-{{% tab name="JSON Template" %}}
-
-```json {class="line-numbers linkable-line-numbers"}
-// "components": [ board & motor ... ],
-"remotes": [
-    {
-      "name": "my-sensor-server",
-      "insecure": true,
-      "address": "localhost:9090"
-    }
-]
-```
-
-{{% /tab %}}
-{{% tab name="Config Builder" %}}
-
-![Creation of a custom sensor remote in the Viam app config builder."](/tutorials/plant-watering-pi/sensor-remote-config-builder.png)
-
-{{% /tab %}}
-{{< /tabs >}}
-
-{{% alert title="Important" color="note" %}}
-
-If you use the Config Builder to make your remote, make sure you have still added the line: `"insecure": true,` to the Raw JSON.
-This line determines that your server does need an SSL certificate.
-
-{{% /alert %}}
-
-#### Add the `MoistureSensor` Process
-
-Then, navigate to the **Processes** subtab and create a process called `run-sensor-server`:
-
-{{< tabs >}}
-{{% tab name="JSON Template" %}}
-
-```json {class="line-numbers linkable-line-numbers"}
 {
-  // "components": [ board & motor ... ] ,
-  // "remotes" [ my-sensor-server ... ] ,
-  "processes": [
-    {
-      "id": "run-sensor-server",
-      "log": true,
-      "name": "sudo",
-      "args": ["-u", "<my_username>", "python", "sensor.py"],
-      "cwd": "/home/<my_username>/plant-watering-robot"
-    }
-  ]
+  "channel_map": {
+    "moisture": 0
+  },
+  "sensor_pin": 8
 }
-```
-
-{{% /tab %}}
-{{% tab name="Config Builder" %}}
-
-![Creation of a process to run the sensor remote server in the Viam app config builder.](/tutorials/plant-watering-pi/sensor-process-config-builder.png)
-
-{{% /tab %}}
-{{< /tabs >}}
 
-Modify the `"args"` and `"cwd"` to match the path to your `plant-watering-robot` directory on your Pi.
-
-Click **Save config**.
-Now, when you navigate to your robot's **Control** tab, you should be able to see your sensor readings:
-
-![Readings from a soil moisture sensor in the Viam app CONTROL tab.](/tutorials/plant-watering-pi/sensor-reading-control.png)
-
-{{% alert title="Tip" color="tip" %}}
-
-If you are having trouble seeing your sensor readings, check the **Logs** table and filter by the **error** level to get more information about the issue.
-
-Make sure that you have modified `<my_username>` in the JSON template above to match your username on your Pi.
-You can run `pwd` in your terminal after SSH'ing into your Pi to see what your username is.
-
-{{% /alert %}}
+This module allows you to get multiple readings at the same time from different channels of the ADC sensor. If you wire and configure another sensor, such as temperature sensor on channel 1, you can add the sensor to the channel_map and get a reading from it.
```

Comment:
```suggestion
This module allows you to get multiple readings at the same time from different channels of the ADC sensor.
If you wire and configure another sensor, such as a temperature sensor on channel 1, you can add the sensor to the `"channel_map"` and get a reading from it.
```

Reply:
^ this but also could we add a link to the module code? Something like:

```
{{< alert title="Info" color="info" >}}
If you would like to see how the module works, you can find its code on [GitHub](link to module code).
{{< /alert >}}
```

Reply:
will add!